redwood-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -83,30 +80,30 @@ internal class ViewPortItems(
   }
 
   override fun onModifierUpdated() {}
+}
 
-  // Fetch the item relative to the entire collection view
-  internal fun itemForGlobalIndex(index: Int): ViewPortItem {
-    items.getOrNull(max(index - itemsBefore, 0))?.let {
-      return it
-    }
-
-    // If we don't have a value, fallback to our pools of placeholders
-    val placeholderIndex = index % placeholders.size
-    return ViewPortItem(placeholders[placeholderIndex], containerSize())
-  }
-
-  private fun containerSize(): CValue<CGSize> {
-    return collectionView.frame().useContents { size.readValue() }
-  }
+private fun UICollectionView.size(): CValue<CGSize> {
+  return frame().useContents { size.readValue() }
 }
 
-internal data class ViewPortItem(
+private data class ViewPortItem(
   val widget: Widget<UIView>,
   val size: CValue<CGSize>,
 )
 
 internal open class UIViewLazyList : LazyList<UIView>, ChangeListener {
 
+  // Fetch the item relative to the entire collection view
```

Comment:
I don't want to break your stack, but in the future: [comments should be sentences](https://nedbatchelder.com/blog/201401/comments_should_be_sentences.html)

Reply:
100% agreed. In this case though, I just relocated the comment instead of writing it myself